dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -329,6 +332,46 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       assert_equal [], @csd_teacher_application_with_partner.sanitize_status_timestamp_change_log
     end
 
+    test 'update appends to the timestamp log if fit workshop is changed' do
+      sign_in @program_manager
+      @csp_facilitator_application.update(status_timestamp_change_log: '[]')
+      @csp_facilitator_application.reload
+
+      assert_equal [], @csp_facilitator_application.sanitize_status_timestamp_change_log
+
+      post :update, params: {id: @csp_facilitator_application.id, application: {fit_workshop_id: @fit_workshop.id, status: @csp_facilitator_application.status}}
+      @csp_facilitator_application.reload
+
+      assert_equal [
+        {
+          title: "Fit Workshop: #{@csp_facilitator_application.fit_workshop_date_and_location}",
```

Comment:
ooh, actually, when I log this, I should be logging "Unassigned" based on whether or not the workshop is assigned (if `application.fit_workshop_id` exists), not based on the value of `application.fit_workshop_data_and_location`, because if the city name is missing in the workshop, for example, that will be `nil` but the workshop is assigned.

But to answer your question, currently those methods don't return "Unassigned", they either return the date and location string or nil.